src/angular.bind.js

```diff
@@ -1,6 +1,8 @@
 if (window.angular.bootstrap) {
   //AngularJS is already loaded, so we can return here...
-  console.log('WARNING: Tried to load angular more than once.');
+  if (!!window.console) {
```

Comment:
yeah but my `if` is testing a boolean, not a _something_ that is truthy or falsy

when i read this code
```js
var foo, bar;
...
if (foo) { ... }
if (!!bar) { ... }
```

I assume foo is a `boolean` while bar is a `something`

as i said it's just personal preference

----- 

fun fact with [momentjs](http://momentjs.com/), `moment.duration(0)` is truthy while being equal to 0

```js
moment.duration(0) == 0; // true
moment.duration(0) === 0; // false
!!moment.duration(0); // true
!!0; // false, obviously
```